collections/guides/quickstarts/quickstart_java.md

```diff
@@ -0,0 +1,234 @@
+
+```bash
+javac BasisTheoryQuickstart.java
+java BasisTheoryQuickstart
+```
+
+You will see a response similar to:
+```bash
+> Create a token:
+{"id":"62a71684-f148-424c-88c6-bdb44031357a","tenant_id":"570b53fb-1ecf-4aaf-9cb2-145e13b566a9","type":"token","privacy":{"classification":"general","impact_level":"high","restriction_policy":"redact"},"created_by":"7281f0ef-eafc-455c-bdae-ce6c99ff8268","created_at":"2022-02-17T21:49:29.2596915+00:00"}
+```
+
+🎉🎉🎉 You’ve created a token 🎉🎉🎉
+
+## Step 6: Read back the raw value from Basis Theory
+
+With our value safely stored in a Token, let’s read that value back to our system. To do this, we will make an HTTP GET request to the [Basis Theory Get a Token API](https://docs.basistheory.com/api-reference/#tokens-get-a-token) endpoint and print the response's raw string value.
+
+We are using the `id` property from the previous Token we created to inject the `tokenId` into the Get a token request. Because we are using vanilla Java, we will simply split the response and grab the `id` from the resulting array.
+
+```java
+// Parse token id from response
+var tokenArray = token.split("\"");
+var tokenId = tokenArray[3];
+
+// Get token
+var getRequest = HttpRequest.newBuilder()
+    .uri(URI.create(String.format("https://api.basistheory.com/tokens/%s", tokenId)))
+    .GET()
+    .header("Content-type", "application/json")
+    .header("Accept", "application/json")
+    .header("BT-API-KEY", "key_E6gVJEn1DRT5m6YHs55Dsa")
+    .build();
+
+var getResponse = client.send(getRequest, BodyHandlers.ofString());
+System.out.println(getRequest);
+System.out.println(getResponse.body());
+```
+
+## Step 7: Run the application
+Test the entire tutorial out by running the application:
+
+```bash
+javac BasisTheoryQuickstart.java
+java BasisTheoryQuickstart
+```
+
+🎉🎉🎉 You’ve successfully created a Token for your data and read it back 🎉🎉🎉
+
+```bash
+> Create a token:
+{"id":"62a71684-f148-424c-88c6-bdb44031357a","tenant_id":"570b53fb-1ecf-4aaf-9cb2-145e13b566a9","type":"token","privacy":{"classification":"general","impact_level":"high","restriction_policy":"redact"},"created_by":"7281f0ef-eafc-455c-bdae-ce6c99ff8268","created_at":"2022-02-17T21:49:29.2596915+00:00"}
+Read a token:
+{"id":"62a71684-f148-424c-88c6-bdb44031357a","type":"token","tenant_id":"570b53fb-1ecf-4aaf-9cb2-145e13b566a9","created_by":"7281f0ef-eafc-455c-bdae-ce6c99ff8268","created_at":"2022-02-17T21:49:29.2596915+00:00","privacy":{"classification":"general","impact_level":"high","restriction_policy":"redact"}}
+```
+
+## Put it all together
+This completes the basic ability to secure data with Tokens and retrieve the raw data back from Basis Theory when you need to use the data in your systems. This flow allows you to secure your data at rest, removes the liability of having the data stored in your databases, and frees you from having to worry about complex encryption logic.
```

Comment:
Maybe add a sentence at the end letting them know we also help them securing it on transit when interacting with 3rd parties via our reactors/proxy.